validator/testdata/feature_tests/amp-mathml.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```

Comment:
keep this in the extension folder. Also needs a `.out` file. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/0.1/test/validator-amp-sidebar.html and https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/0.1/test/validator-amp-sidebar.out as a good sample.

Reply:
ok, i have an uncommitted out file that i think was generated when i started, I'll check that and commit it.

Reply:
I added the file, but I am still missing something.

Reply:
See here for an example [extensions](https://github.com/ampproject/amphtml/tree/master/extensions/amp-gfycat/0.1/test) test directory with an .html and .out.